### ISSUE TITLE
MultipleObjectsReturned - get() returned more than one GitRepository -- it returned 2!

### ISSUE THREAD

**Issue by zbohm** [CZ-NIC/djangocms-git-md-page#5]
```
Internal Server Error: /cs/admin/cms/page/edit-plugin/6184/

MultipleObjectsReturned at /cs/admin/cms/page/edit-plugin/6184/
get() returned more than one GitRepository -- it returned 2!

Request Method: POST
Request URL: https://$HOSTNAME/cs/admin/cms/page/edit-plugin/6184/?cms_path=/cs/proki-zdroje-dat-test/
Django Version: 2.2.13
Python Executable: /usr/bin/uwsgi-core
Python Version: 3.8.5

Traceback:

File "lib/python3.8/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "lib/python3.8/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "lib/python3.8/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "lib/python3.8/site-packages/django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "lib/python3.8/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  44.         response = view_func(request, *args, **kwargs)

File "lib/python3.8/site-packages/django/contrib/admin/sites.py" in inner
  223.             return view(request, *args, **kwargs)

File "lib/python3.8/site-packages/django/views/decorators/clickjacking.py" in wrapped_view
  33.         resp = view_func(*args, **kwargs)

File "lib/python3.8/site-packages/cms/admin/placeholderadmin.py" in edit_plugin
  571.         response = plugin_instance.change_view(request, str(plugin_id))

File "lib/python3.8/site-packages/django/contrib/admin/options.py" in change_view
  1648.         return self.changeform_view(request, object_id, form_url, extra_context)

File "lib/python3.8/site-packages/django/utils/decorators.py" in _wrapper
  45.         return bound_method(*args, **kwargs)

File "lib/python3.8/site-packages/django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "lib/python3.8/site-packages/django/contrib/admin/options.py" in changeform_view
  1529.             return self._changeform_view(request, object_id, form_url, extra_context)

File "lib/python3.8/site-packages/django/contrib/admin/options.py" in _changeform_view
  1572.                 self.save_model(request, new_object, form, not add)

File "lib/python3.8/site-packages/cms/plugin_base.py" in save_model
  339.         return super(CMSPluginBase, self).save_model(request, obj, form, change)

File "lib/python3.8/site-packages/django/contrib/admin/options.py" in save_model
  1088.         obj.save()

File "lib/python3.8/site-packages/git_md_page/models/git_plugins.py" in save
  51.             repo_update.send(sender=self.__class__, instance=self)

File "lib/python3.8/site-packages/django/dispatch/dispatcher.py" in send
  173.         return [

File "lib/python3.8/site-packages/django/dispatch/dispatcher.py" in <listcomp>
  174.             (receiver, receiver(signal=self, sender=sender, **named))

File "lib/python3.8/site-packages/git_md_page/signals/git_update.py" in repository_update
  24.         repository = GitRepository.objects.get(URL=url)

File "lib/python3.8/site-packages/django/db/models/manager.py" in manager_method
  82.                 return getattr(self.get_queryset(), name)(*args, **kwargs)

File "lib/python3.8/site-packages/django/db/models/query.py" in get
  410.         raise self.model.MultipleObjectsReturned(

Exception Type: MultipleObjectsReturned at /cs/admin/cms/page/edit-plugin/6184/
Exception Value: get() returned more than one GitRepository -- it returned 2!
```

**Comment by e3rd** [comment]
I've added two repositories of the same name. When I tried to add a file name, it ended with 500. When I removed one of the repositories, the error vanished.

**Comment by zbohm** [comment]
Fixed in version 2.0.0.
